wazo_auth/token.py

```diff
@@ -189,5 +189,6 @@ def _publish_event(self, tokens, sessions, event_class):
                 logger.warning(
                     'session without token associated: {}'.format(session['uuid'])
                 )
-
-            self._bus_publisher.publish(event_class(**event_args))
+            tenant_uuid = event_args.get('tenant_uuid', None)
+            headers = {'tenant_uuid': tenant_uuid} if tenant_uuid else None
```

Comment:
Don't use `get` when `tenant_uuid` is always defined in `event_args`

```suggestion
            tenant_uuid = event_args['tenant_uuid']
            headers = {'tenant_uuid': tenant_uuid} if tenant_uuid else None
```
or
```suggestion
            headers = None
            if event_args['tenant_uuid']:
                headers = {'tenant_uuid': event_args['tenant_uuid']}
```